rollup.config.mjs

```diff
@@ -0,0 +1,20 @@
+import { dts } from 'rollup-plugin-dts'
+
+const config = [
+  {
+    input: 'projects/ngx-meta/dist/json-ld/index.d.ts',
+    output: {
+      format: 'es',
+      file: 'projects/ngx-meta/dist/json-ld/bundled.d.ts',
+    },
```

Comment:
We'd have to see if the plugin supports multiple inputs & output files by name. Some quick tests seemed to indicate it doesn't. So the other alternative would be to call `rollup` using it's JS API.